client_test.go

```diff
@@ -84,6 +86,14 @@ func buildConcurrentServer(code int, t *testing.T, a func(r *http.Request) []byt
 	}))
 }
 
+func isKnoxDaemonRunning() bool {
+	cmd := exec.Command("pgrep", "knox")
```

Comment:
This seems prone to false positives, i.e. if `knox` is running as part of a test. Maybe make this more specific to the daemon by checking if (1) we're on Linux, and (2) daemon is running via systemctl?